docs/imaging/dlc.rst

```diff
@@ -0,0 +1,114 @@
+.. _Imaging dlc:
+
+=========================
+Imaging: DLC
+=========================
+
+`DeepLabCut <http://www.mackenziemathislab.org/deeplabcut>`_ is a software package for inferring motion capture information from tracking video. Whereas Optitrack makes use of one infra-red reflectors attached to the subject(s), DeepLabCut instead uses a deep neural network trained on human-labelled data of the subject(s), allowing for less interference with subject behaviour. 
+
+
+
+Schema
+------------
+
+The DLC implementation in the imaging pipeline is shown below
+
+.. figure:: /_static/imaging/dlc_schema.png
+   :alt: DLC in the Imaging schema
+   
+   DLC in the Imaging schema
+
+* ``DLCTrackingType``
+
+  Keeps track of the kinds of tracking to which DLC can be assigned. At present, DLC data is only supported for *openfield* tasks
```

Comment:
Maybe: Defines whether the output of a `DLCTrackingProcessingMethod` (see below) is processed further as, for example "Open field" data (`OpenField`).  For example, in the `OpenField` case, the tracking data would be post-processed to extract x/y position, head direction, speed  in relation to the physical environment the animal was tracked in.

Reply:
(This is to distinguish between the "post-" processing and just ingestion of DLC data, which will find its way (no matter what) into `TrackingRaw.DLCPart` and `Tracking.DLCPart` (maybe spell that out).